Dockerfile

```diff
@@ -28,10 +28,15 @@ RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${GOBIN}/${PROJECT_NAME} \
 # =============================================================================
 FROM alpine:3.7 AS final
 
+RUN apk add --update \
+    sudo \
+    libcap
+
 ARG PROJECT_NAME=pravega-operator
 ARG REPO_PATH=github.com/pravega/$PROJECT_NAME
 
 COPY --from=go-builder ${GOBIN}/${PROJECT_NAME} /usr/local/bin/${PROJECT_NAME}
+RUN sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/${PROJECT_NAME}
```

Comment:
@Tristan1900 can you explain why we need the additional dependencies and `setcap` command?

Reply:
Sure! Webhook server needs to listen to 443 port for any requests from Apiserver. By default, it seems the operator has no permission to use the restricted port. `setcap` can grant permission that the operator needs.